science/gr-pcap/Portfile

```diff
@@ -33,7 +33,7 @@ subport gr37-pcap {
 }
 
 depends_lib-append \
-    path:bin/scapy:scapy
+    path:bin/scapy:py-scapy
```

Comment:
I am not convinced this will work as intended. Firstly, one shouldn't depend on a stub port and secondly, the new pyXY-scapy ports will likely not install the "scapy" binary file in ${prefix}/bin. If they do that is a problem as then the different pyXY subports will conflict.

Reply:
You were right.

Reply:
`gnuradio` PG uses `active_python_version_no_dot` which allows to simplify it.

Reply:
And base on https://github.com/search?q=repo%3Aosh%2Fgr-pcap%20scapy&type=code it doesn't use `scapy` as binary, only as python dependency